unitTests/ui/NoNameFacet/NoNameFacetHeaderTest.ts

```diff
@@ -79,5 +79,46 @@ export function NoNameFacetHeaderTest() {
 
       expect($$(operatorElement).findClass('coveo-or')).toBeTruthy();
     });
+
+    it(`when clicking on the toggle Operator button
+      should switch the option correctly`, () => {
+      baseOptions.rootFacetOptions.enableTogglingOperator = true;
+      initializeComponent();
+      const operatorElement = $$(noNameFacetHeader.element).find('.coveo-facet-header-operator');
+      $$(operatorElement).trigger('click');
+
+      expect($$(operatorElement).findClass('coveo-or')).toBeTruthy();
+    });
+
+    it(`when passing the option enableTogglingCollapse (true)
+      should display the accessible Collapse button`, () => {
+      baseOptions.rootFacetOptions.enableTogglingCollapse = true;
+      initializeComponent();
+      const collapseElement = $$(noNameFacetHeader.element).find('.coveo-facet-header-collapse');
+
+      expect($$(collapseElement).getAttribute('aria-label')).toBeTruthy();
+      expect($$(collapseElement).getAttribute('title')).toBeTruthy();
+      expect($$(collapseElement).findClass('coveo-facet-settings-section-show-svg')).toBeTruthy();
+    });
+
+    it(`when passing the option enableTogglingOperator (true) & useAnd (false)
```

Comment:
Hmm, the test description does not seem to match the test?

Reply:
woups copy pasted too fast